dictdatabase/io_unsafe.py

```diff
@@ -1,10 +1,19 @@
 from __future__ import annotations
-from typing import Tuple
+
+import hashlib
+import json
 from dataclasses import dataclass
+from typing import Tuple
+
 import orjson
-import json
-import hashlib
-from . import config, utils, byte_codes, indexing, io_bytes
+
+from . import byte_codes
+from . import config
+from . import indexing
+from . import io_bytes
+from . import utils
+from .index_manager import IndexManager
```

Comment:
I don’t think this needs to be a class, my testing showed that classes add significant overhead over function calls